ui/pages/confirmations/components/confirm/blockaid-loading-indicator/blockaid-loading-indicator.test.tsx

```diff
@@ -8,8 +8,8 @@ import {
 import mockState from '../../../../../../test/data/mock-state.json';
 import { BlockaidResultType } from '../../../../../../shared/constants/security-provider';
 import { renderWithProvider } from '../../../../../../test/lib/render-helpers';
-import { SecurityAlertResponse } from '../../../types/confirm';
 
+import { SecurityAlertResponse } from '../../../../../../app/scripts/lib/ppom/types';
```

Comment:
Ideally, we want to avoid referencing background types in the frontend.

Could we instead update the shared type?

Reply:
Make sense, I reverted this change and updated the UI type.